src/components/settings/SettingsLMSTab/LMSConfigs/utils.tsx

```diff
@@ -174,12 +175,11 @@ export async function handleSaveHelper(
   return !err;
 }
 
-export function checkForDuplicateNames(
-  existingConfigNames: string[], existingData: { displayName: string }): FormFieldValidation {
-  return {
+export function checkForDuplicateNames(existingConfigNames: string[]): FormFieldValidation {
+    return {
     formFieldId: 'displayName',
-    validator: () => {
-      return existingConfigNames?.includes(existingData.displayName)
+    validator: (fields) => {
+      return existingConfigNames?.includes(fields['displayName'])
```

Comment:
If we're editing an existing config, this is always going to trigger, preventing us from saving changes.